src/src/settings.py

```diff
@@ -20,12 +20,15 @@
 TEMPLATE_DIR = os.path.join(APP_DIR, 'templates')
 STATIC_PATH = os.path.join(APP_DIR,'static')
 
-DEV_REPO_URL = 'https://api.github.com/repos/spdx/TEST-LicenseList-XML/issues'
-PROD_REPO_URL = 'https://api.github.com/repos/spdx/license-list-XML/issues'
+LICENSE_REPO_NAME = "license-list-XML"
+LICENSE_TEST_REPO_NAME = "TEST-LicenseList-XML"
+DEV_REPO_URL = 'https://api.github.com/repos/spdx/{0}'.format(LICENSE_TEST_REPO_NAME)
+PROD_REPO_URL = 'https://api.github.com/repos/spdx/{0}'.format(LICENSE_REPO_NAME)
 REPO_URL = DEV_REPO_URL
 
-NAMESPACE_DEV_REPO_URL = 'https://api.github.com/repos/tjasmith/license-namespace-list/issues'
-NAMESPACE_PROD_REPO_URL = 'https://api.github.com/repos/tjasmith/license-namespace-list/issues'
+NAMESPACE_REPO_NAME = "license-namespace-list"
+NAMESPACE_DEV_REPO_URL = 'https://api.github.com/repos/tjasmith/{0}'.format(NAMESPACE_REPO_NAME)
```

Comment:
Should I create a test repo in SPDX for this?

Reply:
Yes,

Reply:
I created to repos and added tjasmith with write access:

- https://github.com/spdx/license-namespace - for production
- https://github.com/spdx/license-namespace-test - for test